modules/030-cloud-provider-yandex/templates/cni.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
   namespace: kube-system
   {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
 data:
-  cni: {{ b64enc "simple-bridge" | quote }}
+  cni: {{ b64enc "cilium" | quote }}
```

Comment:
Are you sure?

Reply:
wow, this comes with one of automatic merge of main branch 😮 
fixed